### PR TITLE
RD-1867 CLI: use and update a single profile

### PR DESCRIPTION
**Pull request by tehasdf**
Instead of always `cfy prof use`, make a single profile with a constant name,
and `cfy prof set` it with the changed values.

Also see the jira for additional details, and see the commit messages as well.

### CODE REVIEW

**Comment by tehasdf** [comment]
needs cloudify-cosmo/cloudify-cli#1259

File: cfy_manager/components/cli/cli.py [hunk 1]
```diff
@@ -33,6 +34,17 @@
 PROFILE_NAME = 'manager-local'
 
 
+@contextmanager
+def _hide_logs():
+    """Increase the logging level, so that the password isn't in the logfile"""
+    current_level = get_file_handlers_level()
+    set_file_handlers_level(logging.ERROR)
```
Comment (geokala) [inline cfy_manager/components/cli/cli.py:41]: Not something to change in this PR, but... this is really not good, we need to do something better about this.

Comment (geokala) [reply cfy_manager/components/cli/cli.py:41]: (and I know it's just copied from elsewhere here, just noting it as it rather draws attention)

Comment (tehasdf) [reply cfy_manager/components/cli/cli.py:41]: hm, change how?

**Review by geokala — APPROVED** [review]

**Pull request merged**